policy/convertZernikesToPsfWidthSetting.yaml

```diff
@@ -0,0 +1,74 @@
+# conversion factors for ts.phosim.utils.ConvertZernikesToPsfWidth
+# units: arcsec of FWHM quadrature contribution per micron of wavefront amplitude
+# the conversion factors below were calculated by Josh Meyers.
```

Comment:
Would also be nice to make sure we have all cameras covered. How hard would it be to add ComCam, LsstFamCam coverage?

Reply:
@jmeyers314 can you answer these two questions? There are the values to convert the Zernike amplitudes from microns to their FWHM contributions.

Reply:
Here's a gist with the computation: https://gist.github.com/jmeyers314/38db2fd38b064be1e12c2992ea9b3e2d

Reply:
Looks like the values for AuxTel are a little bit different.  I wonder if this is confusion about the inner/outer pupil radii again.  I'm _pretty_ sure the values in the gist are correct, but somebody should double check me on those.

Given that the code is pretty short, and doesn't contain any new dependencies, maybe it'd be better to compute these coefficients on-the-fly rather than making a yaml file?  OTOH, the yaml file is nice as a quick reference for the relative influences of different Zernikes, so I'm not sure what's best.

Reply:
I vote to compute the coefficients on-the-fly and the `yaml` file only needs to keep the radius of different telescopes.

Reply:
I am using the [`Instrument` class from `ts_wep`](https://github.com/lsst-ts/ts_wep/blob/92110cfdaaacfcef9a9821cf4cb920be2c7d9487/python/lsst/ts/wep/cwfs/Instrument.py#L45) to pull out the radius and obscuration. You call `instrument.config(camType, dimOfDonutImgOnSensor)` to set up the instrument (`dimOfDonutImgOnSensor` is irrelevant for my purposes). 

This works fine for LsstCam, LsstFamCam, and ComCam. However, this fails for AuxTel because one of the optional arguments is `announcedDefocalDisInMm`, which must match one of the keys in the `offset` section of the instrument yaml file (for example, see the list of possible LsstCam offsets [here](https://github.com/lsst-ts/ts_wep/blob/develop/policy/cwfs/instData/lsst/instParam.yaml)).

This works for LsstCam, LsstFamCam, and ComCam because the default value is 1.5, and each of these instruments have 1.5mm as a potential offset in their yaml files. However, this fails for AuxTel, because the only option for AuxTel is 0.8mm (you can see this [here](https://github.com/lsst-ts/ts_wep/blob/develop/policy/cwfs/instData/auxTel/instParam.yaml)).

There are several ways to get around this:
- I could hard code that when `camType==AuxTel`, I should use `announcedDefocalDisInMm=0.8` when configuring the instrument.
- But If I am going to start hard coding parameters like this, why not just hardcode the radii and obscurations?
- Or I could just not use the `Instrument` class, and directly pull the values I want out of the yaml files.

Any thoughts on what I should do here?

Reply:
@jbkalmbach is trying to solve this here:
https://github.com/lsst-ts/ts_wep/pull/95

Reply:
Great! I will wait on this until the new `Instrument` class/configs have settled.

Reply:
I think `ts_wep` version 2.7 and higher should be able to handle what you want to do now.

Reply:
Great, thanks!